acme.sh

```diff
@@ -265,6 +265,37 @@ _usage() {
   printf "\n" >&2
 }
 
+__debug_bash_helper() {
+  # At this point only do for --debug 3
+  if [ "${DEBUG:-$DEBUG_LEVEL_NONE}" -lt "$DEBUG_LEVEL_3" ]; then
```

Comment:
we don't need to check  DEBUG value  here.

Reply:
The reason I did it that was is to that when someone selects "--debug 3" then all the debug lines will get the filename, function name and line no. If we do it the other way then the "debug 1" and "debug 2" lines will never have the filename, function name, and line no even when debug 3 is used.